intelmq/bots/BOTS

```diff
@@ -653,6 +103,45 @@
                 "xmpp_server": "xmpp server",
                 "xmpp_user": "xmpp username"
             }
+        },
+        "AlienVault OTX": {
```

Comment:
Yes, but if you do not sort, then Python outputs the content in an arbitrary order (which is not the same as the input file) and then you cannot compare it to the original file.